libminifi/src/RemoteProcessorGroupPort.cpp

```diff
@@ -292,6 +292,7 @@ std::pair<std::string, int> RemoteProcessorGroupPort::refreshRemoteSite2SiteInfo
     if (!token.empty())
       client->setRequestHeader("Authorization", token);
 
+    client->setVerbose(false);
```

Comment:
I'd add a comment clarifying that these logs go to the debug level.
```suggestion
    // verbose logs are logged at debug level
    client->setVerbose(/* use_stderr: */ false);
```